Source/InMobiAdapter.swift

```diff
@@ -149,7 +149,13 @@ final class InMobiAdapter: NSObject, PartnerAdapter {
             return .loadFailureAborted
         case .droppingNetworkRequest:
             return .loadFailureNetworkingError
-        @unknown default:
+        case .incorrectPlacementID:
+            return .loadFailureInvalidPartnerPlacement
+        case .sdkNotInitialised:
+            return .loadFailurePartnerNotInitialized
+        case .invalidBannerframe:
+            return .loadFailureInvalidBannerSize
+        default:
```

Comment:
For some reason the `@unknown default` was causing a warning:

> warning: case is already handled by previous patterns; consider removing it

Removing `@default` seems to fix it.

Reply:
Probably because "the compiler will produce a warning if all known elements of the enum have not yet been matched" as discussed in this article https://www.avanderlee.com/swift/unknown-default-enums-in-swift/

Reply:
Ah, it actually looks like this is because the enum is marked as `@frozen`. https://jllnmercier.medium.com/swift-unknown-and-frozen-attributes-8d4eea52d5ac